solutions/dashboard/build/docker-compose.yml

```diff
@@ -17,7 +17,7 @@
 # please email opensource@seagate.com or cortx-questions@seagate.com.
 #
 
-version: "3"
+version: "4"
```

Comment:
Do we need a specific `docker compose` version to use version 4.

Reply:
No, I don't think we need any specific docker-compose version. Actually, I thought it is an incremental version for image generation. That's why I modified the version to 4.

Reply:
It happened by mistake. I will revert it back to 3

Reply:
Done. Updated the docker-compose version to 3